app/server/api/transfer_account_api.py

```diff
@@ -49,9 +47,9 @@ def get(self, transfer_account_id):
             base_query = TransferAccount.query.filter(TransferAccount.is_ghost != True)
 
             if account_type_filter == 'vendor':
-                transfer_accounts_query = base_query.filter_by(has_vendor_role=True)
+                transfer_accounts_query = base_query.filter_by(is_vendor=True)
             elif account_type_filter == 'beneficiary':
-                transfer_accounts_query = base_query.filter_by(has_vendor_role=False)
+                transfer_accounts_query = base_query.filter_by(is_beneficiary=True)
```

Comment:
this was causing an error when I wrote a test for it so change to `is_vendor` and `is_beneficiary`